models/repo.go

```diff
@@ -1699,6 +1699,12 @@ func UpdateRepository(repo *Repository, visibilityChanged bool) (err error) {
 	return sess.Commit()
 }
 
+// UpdateRepositoryUpdatedTime updates a repository's updated time
+func UpdateRepositoryUpdatedTime(repoID int64, updateTime time.Time) error {
```

Comment:
Where does this function come from?

Reply:
Come from https://github.com/go-gitea/gitea/pull/7486/files#diff-6b299323ba0a072129c7841649f49536L431

Reply:
Ah ok, fair enough.